tests/framework/user_guide/ForwardSamplingStrategies/forwardSamplingSparseGrid.xml

```diff
@@ -181,17 +181,21 @@
       <Sampler class="Samplers" type="SparseGridCollocation">SG</Sampler>
       <Output class="DataObjects" type="HistorySet">histories</Output>
     </MultiRun>
+    <RomTrainer name="train">
+      <Input class="DataObjects" type="HistorySet">histories</Input>
+      <Output class="Models" type="ROM">rom</Output>
+    </RomTrainer>
```

Comment:
Why this change? It's not a problem, just curious.

Reply:
Just want to get a better understanding of the steps, I put this in sequence order, even though it does not to.